qdrant-landing/content/documentation/faq/qdrant-fundamentals.md

```diff
@@ -81,3 +81,35 @@ We only guarantee compatibility if you update between consecutive versions. You
 
 In case your version is older, we only guarantee compatibility between two consecutive minor versions. This also applies to client versions. Ensure your client version is never more than one minor version away from your cluster version.
 While we will assist with break/fix troubleshooting of issues and errors specific to our products, Qdrant is not accountable for reviewing, writing (or rewriting), or debugging custom code.
+
+### Which embedding models does FastEmbed support, and how does it utilize memory?
+
+You can find the list of supported models [here](https://qdrant.github.io/fastembed/examples/Supported_Models/).
+
+Memory usage in fastEmbed depends on several factors:
+1. **Size of the Text**: Everything, including the text and its vectors, is loaded into memory. As vectors are computed, RAM consumption increases accordingly.
```

Comment:
this is not true, fastembed is able to stream encoding, it doesn't need all vectors and texts in ram